packages/adapter-static/README.md

```diff
@@ -38,6 +38,25 @@ The directory to write static assets (the contents of `static`, plus client-side
 
 Specify a fallback page for SPA mode, e.g. `index.html` or `200.html` or `404.html`.
 
+### entry_file_names
```

Comment:
we use underscores for internal variable names and `camelCase` for public APIs

I think that using "entry" in the name here could be confused with the crawl entry points especially after https://github.com/sveltejs/kit/issues/2185 is implemented

I think a name like `outputFileName` might be a bit less likely to cause confusion